### PR TITLE
added support for tags in java sdk - no refactoring for now

### PR DESCRIPTION
**Pull request by dineshrathi**

### CODE REVIEW

File: src/main/java/com/qubole/qds/sdk/java/api/HiveCommandBuilder.java [hunk 1]
```diff
@@ -25,4 +25,6 @@
     public HiveCommandBuilder clusterLabel(String clusterLabel);
 
     public HiveCommandBuilder name(String queryName);
+
+    public HiveCommandBuilder tags(String[] tags);
```
Comment (mindprince) [inline src/main/java/com/qubole/qds/sdk/java/api/HiveCommandBuilder.java]: Nit: You have named the argument `tags` at some places and `queryTags` at other places.

File: src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+import com.qubole.qds.sdk.java.api.HadoopCommandBuilder;
+import com.qubole.qds.sdk.java.client.*;
+import com.qubole.qds.sdk.java.entities.CommandResponse;
+import com.qubole.qds.sdk.java.entities.ResultValue;
+
+/**
+ * Created by Dinesh on 24-12-2014.
+ */
+public class TestHadoopCommand
```
Comment (mindprince) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: I don't understand this - what is this testing? Seems more like an example.

File: src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
+package com.qubole.qds.sdk.java;
+
+import com.qubole.qds.sdk.java.api.HadoopCommandBuilder;
+import com.qubole.qds.sdk.java.client.*;
+import com.qubole.qds.sdk.java.entities.CommandResponse;
+import com.qubole.qds.sdk.java.entities.ResultValue;
+
+/**
+ * Created by Dinesh on 24-12-2014.
```
Comment (mindprince) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: Remove this.

File: src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java (hunk 2 above)
Comment (dineshrathi) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: yes, its an example, one customer asked for a hadoop example on java sdk so i added parallel to existing example, also added tags usage for it. is it okay?

File: src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java (hunk 3 above)
Comment (dineshrathi) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: done

File: src/main/java/com/qubole/qds/sdk/java/api/HiveCommandBuilder.java (hunk 1 above)
Comment (dineshrathi) [inline src/main/java/com/qubole/qds/sdk/java/api/HiveCommandBuilder.java]: fixed

File: src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java (hunk 2 above)
Comment (mindprince) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: In a file named TestXY, I usually expect a test and not an example.

Let's either move this example inside Tester.java or better yet move both these examples in a new examples/ folder. We can do this as part of a separate PR. In any case TestXY shouldn't contain an example.

Comment (qubole) [inline src/test/java/com/qubole/qds/sdk/java/TestHadoopCommand.java]: will do in next pr.

**Comment by mindprince** [comment]
Merged.
